openfisca_core/taxbenefitsystems.py

```diff
@@ -232,6 +235,38 @@ def load_extension(self, extension):
         if path.isfile(param_file):
             self.add_legislation_params(param_file)
 
+    def apply_reform(self, reform_path):
+        """
+        Generates a new tax and benefit system applying a reform to the tax and benefit system.
+
+        The current tax and benefit system is **not** mutated.
+
+        :param string reform_path: The reform to apply. Must respect the format *installed_package.sub_module.reform*
+
+        :returns: A reformed tax and benefit system.
+
+        Exemple:
+
+        >>> self.apply_reform('openfisca_france.reforms.inversion_revenus')
+
+        """
+        from reforms import Reform
+        try:
+            reform_package, reform_name = reform_path.rsplit('.', 1)
+        except ValueError:
+            raise ValueError(u'`{}` does not seem to be a path pointing to a reform. A path looks like `some_country_package.reforms.some_reform.`'.format(reform_path))
```

Comment:
👍 

I promised  myself I'd be more careful about this, but I forgot as I just moved this code around